test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -230,7 +247,8 @@ private HapiApiSpec runAccountBalancesClientSaveLoadTest() {
 			@Override
 			public Optional<HapiSpecOperation> get() {
 				int next;
-				if ((next = createdSofar.getAndIncrement()) >= totalTestTokens) {
+				next = createdSofar.getAndIncrement();
```

Comment:
nit: same here, should be in a single line as in my original comment.

Reply:
changed.